graphql/database/cause/causes/cats/impact.claimImpactSubtitle.md

```diff
@@ -1 +1 @@
-##### You did it! You just turned your tab into a treat for a cat. Keep it up!
+##### You did it! You just turned your tab into a treat for a cat. Keep it up, and do good with every new tab!
```

Comment:
I find the addition pretty redundant and had already cut it from the seas copy. I think leave as-is.